.clang-tidy

```diff
@@ -21,6 +21,10 @@ WarningsAsErrors: '*'
 #       inst_id = name_ref->value_id;
 #                 ^ unchecked access to optional value
 #     }
+# - google-readability-casting is deeply broken, and fires regularly on calls
+#   to constructors. Worse, it suggests switching to a call to a constructor
+#   which doesn't fix the warning. This has been disabled by many major users
+#   of clang-tidy (Chromium, etc).
```

Comment:
Note, the way you're talking about this issue is pretty strong. Would it be worth softening up the wording for other readers? e.g., maybe we could focus on the issue as it affects us:

```suggestion
# - google-readability-casting fires regularly on calls to constructors. It suggests switching to a call to a constructor, which doesn't fix the warning.
```